library/src/app/components/trade/trade.component.html

```diff
@@ -0,0 +1,117 @@
+<div
+  [ngClass]="{
+    'cybrid-dark-theme': (configService.config$ | async)?.theme === 'DARK'
+  }"
+>
+  <ng-container *ngIf="(isRecoverable$ | async) !== false; else error">
+    <ng-container *ngIf="(isLoading$ | async) !== true">
+      <mat-tab-group animationDuration="0ms">
+        <mat-tab label="Buy">
+          <ng-container [formGroup]="quoteGroup" *ngIf="quote$ | async as quote">
+            <div fxLayout="column" fxLayoutAlign="center" class="tab-wrapper">
+              <mat-form-field appearance="outline">
+                <mat-label>
+                  <span>{{ 'trade.amount' | translate }}</span>
+                  <span class="code">
+                {{
+                    input === 'fiat' ? quote.asset.code : quote.counterAsset.code
```

Comment:
Is `input` the side of the trade, i.e., asset or counterasset?

Reply:
Actually it's independent from it. It's just to swap the units on the input. I can buy 1 coin or $10 worth, or I can sell 1 coin or $10 worth.